packages/labs/src/components/popover/popoverMigrations.ts

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
I vote to rename this file to `popoverMigrationUtils.ts`. Suffixing bags of functions with `-Utils.ts` ensures we know exactly what that file is at a glance, and the consistency is nice too.

Reply:
:+1: